src/desktop/apps/articles/templates/meta/news.jade

```diff
@@ -0,0 +1,15 @@
+title Magazine | Artsy
```

Comment:
These should probably be changed to "News | Artsy". It might change down the line but at least it's a more accurate placeholder than Magazine.